ethstore/src/account/safe_account.rs

```diff
@@ -51,6 +51,20 @@ impl From<json::Crypto> for Crypto {
 	}
 }
 
+impl From<json::KeyFile> for SafeAccount {
```

Comment:
This `From` implementation was the reason we had a bug that was creating new keys everytime you run parity.

We have two explicit methods that can be used to instantiate `SafeAccount`

`SafeAccount::create(...)` - when creating new account (that haven't been saved to disk yet)
`SafeAccount::from_file(KeyFile, filename: String)` - when reading KeyFile from existing file on disk

I think that we should use only those, cause you should make a decision with regard to filename.